doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -476,14 +476,28 @@ In Ltac2, we have the following backtracking primitives, defined in the
   val plus : (unit -> 'a) -> (exn -> 'a) -> 'a
   val case : (unit -> 'a) -> ('a * (exn -> 'a)) result
 
-If one views thunks as lazy lists, then `zero` is the empty list and `plus` is
-list concatenation, while `case` is pattern-matching.
+Informally, the primitive `zero e` ends the current computation and returns to the last backtracking point with exception `e`.
+The primitive `plus t handler` installs a backtracking point around `t`, using `handler` to handle any exception raised by a `zero` while evaluating `t`.
```

Comment:
How is that different from raising and catching exception ?

Reply:
Exception-raising code can have at most one success, the tactic monad is multisuccess. The intuitive model is really that it's a stream (where exceptions are options).